crates/ruff_cli/tests/integration_test.rs

```diff
@@ -598,6 +598,7 @@ fn stdin_format_jupyter() {
     }
 
     ----- stderr -----
+    warning: Ignoring file . in favor of standard input.
```

Comment:
I'm not enthused about this change. I'm unsure how best to fix it. I'm not sure I want to deal with changing the default for `files` later so we can detect this case but it also feels kind of wrong to just avoid raising the warning if the file is `.`.

Source: https://github.com/astral-sh/ruff/blob/7b4b0045067b8eb41d0b03d9c3549ce3595373fa/crates/ruff_cli/src/lib.rs#L87-L95

Reply:
:/ bde7712ad